src/Environment.php

```diff
@@ -115,6 +118,7 @@ public function __construct(LoaderInterface $loader, $options = [])
         $this->strictVariables = (bool) $options['strict_variables'];
         $this->setCache($options['cache']);
         $this->extensionSet = new ExtensionSet();
+        $this->globals = $options['globals'];
```

Comment:
this does not ensure anymore that `$this->globals` is an array, as it accepts any incoming values.

Reply:
And if i replace :
`$this->globals = $options['globals'];`
by : 
`$this->globals = (is_array($options['globals']))? $options['globals'] : [] ;`